src/app/space/add-app-overlay/add-app-overlay.component.ts

```diff
@@ -23,14 +23,15 @@ import { Application, DeploymentApiService } from '../create/deployments/service
 })
 export class AddAppOverlayComponent implements OnDestroy, OnInit {
   currentSpace: Space;
-  isProjectNameValid: boolean;
+  isProjectNameValid = false;
```

Comment:
why did you remove the type?

Reply:
when we assign a primitive value, type is not needed by default. Type is defined from the primitive value we assign

Reply:
I prefer the verbose style but yes you are right, that works too.